lib/pay_day_loan.ex

```diff
@@ -182,11 +279,21 @@ defmodule PayDayLoan do
 
   @doc """
   Check for cached pid, but do not request a load
+
+  This is a legacy API method and may be deprecated.  Use `peek/2`.
   """
   @spec peek_pid(pdl :: t, key :: PayDayLoan.key)
   :: {:ok, pid} | {:error, :not_found}
   def peek_pid(pdl = %PayDayLoan{}, key) do
-    CacheStateManager.get_pid(pdl.cache_state_manager, key)
+    peek(pdl, key)
```

Comment:
I looked into how to do that at compile time.  It's doable, but not sure it's worth the amount of effort..